great_expectations/render/renderer/slack_renderer.py

```diff
@@ -136,29 +132,38 @@ def render(
         query["blocks"].append(footer_section)
         return query
 
-    def _custom_blocks(self, evr):
-        return None
-
     def _get_report_element(self, docs_link):
-        if docs_link is None:
-            logger.warn("No docs link found. Skipping data docs link in slack message.")
-            return
-
-        if "file://" in docs_link:
-            # handle special case since Slack does not render these links
-            report_element = {
-                "type": "section",
-                "text": {
-                    "type": "mrkdwn",
-                    "text": f"*DataDocs* can be found here: `{docs_link}` \n (Please copy and paste link into a browser to view)\n",
-                },
-            }
+        report_element = None
+        if docs_link:
+            try:
+                if "file://" in docs_link:
+                    # handle special case since Slack does not render these links
+                    report_element = {
+                        "type": "section",
+                        "text": {
+                            "type": "mrkdwn",
+                            "text": f"*DataDocs* can be found here: `{docs_link}` \n (Please copy and paste link into a browser to view)\n",
+                        },
+                    }
+                else:
+                    report_element = {
+                        "type": "section",
+                        "text": {
+                            "type": "mrkdwn",
+                            "text": f"*DataDocs* can be found here: <{docs_link}|{docs_link}>",
+                        },
+                    }
+            except:
+                logger.warning(
+                    f"""SlackRenderer had a problem with generating the docs link.
+                        link used to generate the docs link is: {docs_link} and is of type: {type(docs_link)}
+                    """
+                )
+                return
```

Comment:
This is the informative log message that I added after our discussion today @eugmandel